lang/image.bzl

```
@@ -263,11 +275,14 @@ image = struct(
         # The dependency whose runfiles we're appending.
         # If not specified, then the layer will be treated as the top layer,
         # and all remaining deps of "binary" will be added under runfiles.
-        "dep": attr.label(),
+        "dep": attr.label(cfg = container_transition),
         "directory": attr.string(default = "/app"),
         "entrypoint": attr.string_list(default = []),
         "legacy_run_behavior": attr.bool(default = False),
         "workdir": attr.string(default = ""),
+        "_whitelist_function_transition": attr.label(
```

```suggestion
        "_allowlist_function_transition": attr.label(
```

❤️  https://www.theregister.co.uk/2019/09/03/chromium_microsoft_offensive/

The change also helps non-native English speakers comprehend this code.

---

Unfortunately... This name is hardcoded in Bazel, I cannot change it.  This stuff is not even documented.  Even the path "tools/whitelists/function_transition_whitelist" is hardcoded.